portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,42 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Austin Teshuba</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Austin Teshuba</h1>
+
+      <img class="img-primary" src="/images/headshot.jpeg" alt="Profile Picture">
+
+      <p>Hi! Welcome to my portfolio.</p>
+
+      <a href="https://github.com/austinteshuba" target="_blank">Github</a>
+
+      <p> I'm a rising 3rd-year software engineering student at Western University.</p>
```

Comment:
nit: I think you can omit </.p> (https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags)

Reply:
Fixed! Thanks for pointing this out